tools/build/scripts/extract.js

```diff
@@ -8,7 +8,7 @@ const argv = require("yargs").argv;
 const path = require("path");
 const paths = require("./config/paths");
 const fs = require("fs-extra");
-const readDirectory = require("recursive-readdir");
+const readDirectory = require("./utils/recursiveReaddir");
```

Comment:
can you verify the `extract-api` script is working as we expect it to? you should be able to delete the output for some of the existing `extract-api` output, and then rerun the script with your changes from this pr and there should be no diff between the 2.

Reply:
Already done. I both deleted all *.api.md and ran extract-api before and after the changes. There's no differences in the output.